components/bl-list-component/src/list.js

```diff
@@ -0,0 +1,30 @@
+import { ListContent } from './list-content';
+
+export function List ({
+  component, eventHandlers, child
+}) {
+
+  const listType = (type) => {
+    if(type && type === 'ol') {
```

Comment:
Create map for enums

const ListTypes = {
  ORDERED: 'ol',
  UNORDERED: 'ul'
}

Reply:
Also read this article https://www.robinwieruch.de/react-as-prop/

briefly:

```
<List as="ul" ...>
```

and in list component:
```
const Component = as

return (<Component ... />)
```